docs/contributing/editing-docs.md

```diff
@@ -10,15 +10,17 @@ The [Islandora Documentation Interest Group](https://github.com/islandora-intere
 To edit documentation in Islandora, you must:
 
 - have a [GitHub](https://github.com/) account.
-    - NOTE: Github accounts are free and only require a valid email address.
-- have a very basic familiarity with GitHub terms such as [fork](https://guides.github.com/activities/forking/), [branch](https://guides.github.com/activities/hello-world/#branch), and [pull request](https://guides.github.com/activities/hello-world/#pr).
-- be a member of the Islandora [GitHub Organization](https://github.com/orgs/Islandora/people).
-    - This is only needed if you want to commit changes directly to the documentation. Non-members may still make pull requests. To request an invite, email community@islandora.ca or ask on the Islandora Slack.
-- have a basic familiarity with [Markdown](http://en.wikipedia.org/wiki/Markdown), or a good [MarkDown cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
-    - for example, you may use MarkDown syntax to create links and section headings.
+    - Github accounts are free and only require a valid email address.
+- have a very basic familiarity with Git terms such as [fork](https://guides.github.com/activities/forking/), [branch](https://guides.github.com/activities/hello-world/#branch), and [pull request](https://guides.github.com/activities/hello-world/#pr).
+- optionally, be a member of the Islandora [GitHub Organization](https://github.com/orgs/Islandora/people).
+    - To request an invite, email community@islandora.ca or ask on the Islandora Slack.
+    - This is only needed if you want to commit changes directly to the documentation. Non-members may still make pull requests. 
+- be willing to learn with [Markdown](http://en.wikipedia.org/wiki/Markdown) - a good [Markdown cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) can help.
```

Comment:
Did you mean to leave the word "with" in line 18? I can make my own PR to fix that and run a CLI spellchecker while I am at it.